src/views/AddPeopleAndRoles.vue

```diff
@@ -29,10 +29,6 @@ import { PeopleAndRoles } from '@/components/AddPeopleAndRoles'
   }
 })
 export default class AddPeopleAndRoles extends Mixins(EntityFilterMixin) {
-  // Global getters
-  @Getter isEntityType!: GetterIF
```

Comment:
This getter, and some getters and actions in other components, is already declared in the mixin (which is essentially a superclass of this component), so it doesn't need to be declared again. (My VS Code "Vetur" extension complains about re-declarations.)